src/EventListener/InitializeSystemListener.php

```diff
@@ -181,10 +190,7 @@ protected function boot(Request $request = null)
         $this->initializeLegacySessionAccess();
         $this->setDefaultLanguage();
 
-        // Fully load the configuration
-        $objConfig = Config::getInstance();
```

Comment:
> because we cannot foresee the ramifications.

We can. It's a pretty simple piece of code and adapters will always be untestable, that's why we write them.